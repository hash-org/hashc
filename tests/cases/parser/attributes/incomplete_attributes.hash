// stage=parse, run=fail

{
    foo := struct (
        field: i32,
        #invalid
        // ~ERROR: expected struct field here
    )
}


{
    // ~ERROR: expected fn parameter here
    foo := (field: i32, #incomplete) => {}
}




#invalid 
// ~ERROR: expected expression here
