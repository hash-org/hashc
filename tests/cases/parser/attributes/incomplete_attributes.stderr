error: expected a type annotation
 --> $DIR/incomplete_attributes.hash:6:17
5 |           field: i32,
6 |           #invalid
  |                   ^ 
7 |           // ~ERROR: expected struct field here
  = help: expected either `identifier`, `@`, `!`, `#`, `&`, `<`, `(`, `{`, or `[`

error: expected a name here
  --> $DIR/incomplete_attributes.hash:14:36
13 |       // ~ERROR: expected fn parameter here
14 |       foo := (field: i32, #incomplete) => {}
   |                                      ^ 
15 |   }
   = help: expected a `identifier`

error: expected an expression
  --> $DIR/incomplete_attributes.hash:20:9
19 |   
20 |   #invalid 
   |           ^ 
21 |   // ~ERROR: expected expression here
